stable/jupyter/templates/jupyter-configmap.yaml

```diff
@@ -78,6 +78,7 @@ data:
 {{- if and .Values.v3io.tsdbctl .Values.v3io.tsdbctl.download }}
     curl -sSL -o ${IGUAZIO_HOME}/bin/tsdbctl {{ .Values.v3io.tsdbctl.download }}
     chmod +x ${IGUAZIO_HOME}/bin/tsdbctl
+    echo alias tsdb=\"tsdbctl -s {{ .Values.webapi.url }} -k {{ .Values.v3io.accessKey }}\"
```

Comment:
I suggest to change the name of the binary to tsdb-cli (line 79 above) and than the alias can be "tsdbctl"
@bivas any problem with changing the file name?

Reply:
You don't have to rename it. You're allowed to use `alias tsdbctl=\"tsdbctl...`

‼️ There's not value `.Values.webapi` (there _can_ be `.Values.v3io.api.hostname`) - you're going to fail deploy. You better use `${V3IO_API}` (and `${V3IO_ACCESS_KEY}`)
Also, you must wrap this alias definition with a check that those values are set or it will fail during execution.

Reply:
I got `{{ .Values.webapi.url }}` from there https://github.com/v3io/helm-charts/blob/9d89bda/stable/tsdb-functions/templates/query.yaml#L24.

The reason the alias is to a different name is that you might want to specify another server or token.

Reply:
But this values path doesn't exists here. You can check a live deployment and see how this is set.

Reply:
Please check the latest revision. It should address all the above.